vault-consul-acl/step2.md

```diff
@@ -13,7 +12,7 @@ Success! Enabled the consul secrets engine at: consul/
 
 ### Configure Vault to connect and authenticate to Consul
 
-The secrets engine requires a management token to interact with Consul. 
+The secrets engine requires a management token to interact with Consul.
```

Comment:
Below in the "First, create a management .." Could we add that they're creating the token with the global management policy which was generated for the by Consul when ACLs were enabled?